docs/learn/storage/aws-ami.md

```diff
@@ -0,0 +1,95 @@
+# Kaia Endpoint Node AMIs
+
+Kaia offers an AWS AMI ([Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)) for Kaia endpoint nodes (EN). These AMIs are pre-installed with both the EN software and the chaindata, enabling users to quickly and conveniently launch a fully operational node. With Kaia's AMI, setting up a new endpoint node on demand becomes seamless, allowing for rapid scaling and simplifying the process of joining the Kaia network.
+
+## Types of EN AMIs
+
+We provide different types of AMIs for the chaindata synced with different sync modes.
+
+- plain? TBU e.g., `kaia-xxxx-clean-en-xxxx`
```

Comment:
Make it a table like in https://docs.kaia.io/misc/operation/chaindata-snapshot/ this page

Reply:
Done, is that what you asked for?

Reply:
Nice. Thanks!